File: 3110-score-of-a-string/3110-score-of-a-string.py
class Solution(object):
    def scoreOfString(self, s):
        """
        :type s: str
        :rtype: int
        """
        count = 0
        length=len(s)-1
        for i in range(length):
            x = ord(s[i]) - ord(s[i + 1])
            if x < 0:
                x = -x
            count += x
        return count

File: 3110-score-of-a-string/3110-score-of-a-string.cpp
class Solution {
public:
    int scoreOfString(string s) {
        int length=s.size()-1;
        int score=0;
        for(int i=0;i<length;i++){
            int x=s[i]-s[i+1];
            if(x<0){
                x=-x;
            }
            score+=x;
        }
        return score;
    }
};